src/System Application/App/Page Style/app.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id":  "36bf351b-e157-4cf2-932d-b600716b0b4a",
+    "name":  "Page Style",
+    "publisher":  "Microsoft",
+    "brief":  "Provides an enum for defined page styles to set the StyleExpr property",
+    "description":  "Provides an enum for defined page styles to set the StyleExpr property",
+    "version":  "25.0.0.0",
+    "privacyStatement":  "https://go.microsoft.com/fwlink/?linkid=724009",
+    "EULA":  "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help":  "https://go.microsoft.com/fwlink/?linkid=2103698",
+    "url":  "https://go.microsoft.com/fwlink/?linkid=724011",
+    "logo":  "",
+    "dependencies":  [
+        
+                     ],
+    "internalsVisibleTo":  [
+        
+                           ],
+    "screenshots":  [
+
+                    ],
+    "platform":  "24.0.0.0",
+    "target":  "OnPrem",
+    "idRanges":  [
+                     {
+                         "from":  50000,
```

Comment:
I added it as a new app, I did not find any existing app where it was possible to add. Please provide the app where I should add or provide an idRange.

Reply:
I'm also not sure we have a fitting module as of now. But I think a "Control Style" app is too narrow. Let's make a system enums app or something along those lines. The ID  range 1-10 would seem fitting for such an app.

Reply:
Set the id-range from 1-10